src/tailwind.config.js

```diff
@@ -90,6 +90,17 @@ module.exports = {
 			"theme-hint-700": "var(--theme-color-hint-700)",
 			"theme-hint-800": "var(--theme-color-hint-800)",
 			"theme-hint-900": "var(--theme-color-hint-900)",
+
+			"blue-50": "var(--theme-color-blue-50)",
```

Comment:
this is still `blue` and not `info`

Reply:
thanks!

Reply:
actually I'm gonna rename all the references, not just the variables

Reply:
_should_ be good now